src/components/common/ContactInfo.vue

```diff
@@ -1,18 +1,18 @@
 <template>
```

Comment:
This really needs to be a shared component 😁 

(future)

Reply:
Agreed!